src/Bundle/Grid/AbstractGrid.php

```diff
@@ -30,7 +30,7 @@ public function toArray(): array
     private function createGridBuilder(): GridBuilderInterface
     {
         if ($this instanceof ResourceAwareGridInterface) {
-            return GridBuilder::create($this::getName(), $this::getResourceClass());
+            return GridBuilder::create($this::getName(), $this->getResourceClass());
```

Comment:
if I understand correctly, this class(parent of `AbstractGrid`) is already initiated, am I correct?